hbase-client/src/main/java/org/apache/hadoop/hbase/client/ConnectionFactory.java

```diff
@@ -216,21 +217,53 @@ public static Connection createConnection(Configuration conf, User user) throws
    */
   public static Connection createConnection(Configuration conf, ExecutorService pool,
     final User user) throws IOException {
+    return createConnection(conf, pool, user, null);
```

Comment:
Better just use Collections.emptyMap here, so we do not need a null check in later code, usually.